src/melee/ft/chara/ftGigaKoopa/ftgigakoopa.c

```diff
@@ -78,9 +77,9 @@ void func_8014F98C(s32 arg0, s32* arg1, s32* arg2)
     }
 }
 
-extern const s32 lbl_803D3984[];
+/* static */ extern const s32 lbl_803D3984[];
 
-// UB warning: this function may use offset uninitialized
+/// @remarks UB warning: this function may use offset uninitialized
```

Comment:
No idea what "UB" stands for.

Reply:
Undefined Behavior, in the context of the C/C++ standard